drivers/src/bedrock/index.ts

```diff
@@ -89,60 +89,109 @@ export class BedrockDriver extends AbstractDriver<BedrockDriverOptions, BedrockP
         }
     }
 
-    extractDataFromResponse(prompt: BedrockPrompt, response: InvokeModelCommandOutput): Completion {
-
-        const decoder = new TextDecoder();
-        const body = decoder.decode(response.body);
-        const result = JSON.parse(body);
-
-        const getTextAnsStopReason = (): string[] => {
-            if (result.generation) {
-                // LLAMA2
-                return [result.generation, result.stop_reason]; // comes in coirrect format (stop, length)
-            } else if (result.generations) {
-                // Cohere
-                return [result.generations[0].text, cohereFinishReason(result.generations[0].finish_reason)];
-            } else if (result.chat_history) {
-                //Cohere Command R
-                return [result.text, cohereFinishReason(result.finish_reason)];
-            } else if (result.completions) {
-                //A21
-                return [result.completions[0].data?.text, a21FinishReason(result.completions[0].finishReason?.reason)];
-            } else if (result.content) {
-                // Claude
+    //Update this when supporting new models
+    static getTextAnsStopReason(result:any, prompt?: BedrockPrompt): CompletionChunkObject {
```

Comment:
typo :)

Reply:
Name inherited from old function that was refactored. Name now changed to getExtractedCompletionChunk, to be more descriptive of new function and fix type.